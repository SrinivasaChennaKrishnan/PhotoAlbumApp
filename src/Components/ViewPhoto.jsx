import React from 'react';
import Classes from './Styles/CommonStyle.module.css';

function ViewPhoto() {
  return (
    <div className={Classes.center}>
      ViewPhoto
    </div>
  );
}

export default ViewPhoto;
