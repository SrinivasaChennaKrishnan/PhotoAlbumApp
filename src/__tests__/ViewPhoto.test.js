import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ViewPhoto from '../Components/ViewPhoto'

/** Test View Photo Component renders without crashing  */
describe('ViewPhoto', () => {
    it('renders without crashing given the required props', () => {
        const props = {
            albumData: [],
            photolistObject: [],
            loader: false,
            albumMessage: "INITIAL",
            photoListMessage: "INITIAL",
            picSource: "",
            showPhoto: false,
            selectedAlbum: [],
            selectedPhoto: [],
            dispatch: jest.fn()
        }
        const wrapper = shallow(<ViewPhoto {...props} />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})
