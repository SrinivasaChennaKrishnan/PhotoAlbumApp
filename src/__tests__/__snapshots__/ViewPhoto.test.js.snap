// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewPhoto renders without crashing given the required props 1`] = `
<div
  className="center"
>
  <Container
    className="containerOutline"
    fluid={false}
  >
    <ForwardRef
      className="backToAlbumRow"
      noGutters={false}
    >
      <p
        className="backToAlbum"
        onClick={[Function]}
      >
         &lt;&lt; Back to Photo Gallery
      </p>
    </ForwardRef>
    <ForwardRef
      noGutters={false}
    >
      <p
        className="ml20"
      >
        <span
          className="albumDescription"
        >
          Showing the image
           
          <span
            className="albumDescriptionAlbumName"
          />
           
          from the Album
           
        </span>
         
        <span
          className="albumDescriptionAlbumName"
        />
      </p>
    </ForwardRef>
    <Container
      className="albumOutline"
      fluid={false}
    >
      <ForwardRef
        noGutters={false}
      />
    </Container>
  </Container>
</div>
`;
